### ISSUE TITLE
The fourth ConvBlock of the BallDetection forward() function  doesn't use DropOut in file 'models/TTNet.py'

### ISSUE THREAD

**Issue by dozingLee** [maudzung/TTNet-Realtime-for-Table-Tennis-Pytorch#19]
Will this greatly affect the TTNet?

` 
class BallDetection(nn.Module):
    def __init__(self, num_frames_sequence, dropout_p):
        .......

    def forward(self, x):
      x = self.relu(self.batchnorm(self.conv1(x)))
      out_block2 = self.convblock2(self.convblock1(x))
      x = self.dropout2d(out_block2)
      out_block3 = self.convblock3(x)
      out_block4 = self.convblock4(out_block3)
      x = self.dropout2d(out_block4)
    
      # out_block5 = self.convblock5(out_block4) # Original Code
      out_block5 = self.convblock5(x)  # The Code should be this.
      
      features = self.convblock6(out_block5)
      x = self.dropout2d(features)
      
      x = x.contiguous().view(x.size(0), -1)
      x = self.dropout1d(self.relu(self.fc1(x)))
      x = self.dropout1d(self.relu(self.fc2(x)))
      out = self.sigmoid(self.fc3(x))
    
      return out, features, out_block2, out_block3, out_block4, out_block5`
